statement.md

```diff
@@ -0,0 +1,116 @@
+
+* Are typically hard to detect, costly to fix when they appear in production, and often used in exploits;
+
+* Persist despite decades of developer training and tooling aimed at preventing them.
+
+Memory safe languages (e.g. **Rust**, **Swift**, or **Java**) mitigate these risks by enforcing strict rules, making it impossible to introduce entire classes of memory-related bugs by default. This makes memory safety one of the most powerful methods for achieving secure-by-design systems.
+
+# Why Memory Safe Tech Is Crucial
+
+Choosing memory safe technologies is one of the most impactful decisions a manufacturer of digital infrastructure components can make to improve cybersecurity. 
+
+* Security gains are systemic: By transitioning to memory safe languages, up to 70% of vulnerabilities can be eliminated.
+
+* Security becomes scalable: It reduces reliance on human vigilance, which is increasingly unsustainable in the face of talent shortages, increasingly complex systems, and growing attack surfaces.
+
+* Prevention becomes the default: By enforcing security at the language level, the software becomes safer by default, not by afterthought.
+
+# The Urgency for Action: Secure-by-Design
+
+We are at a pivotal moment:
+
+* Global tensions and cyber warfare are on the rise, while our dependence on digital infrastructure grows daily;
+
+* The cost of cyber breaches continues to climb, with some incidents costing billions;
+
+* There is an acute shortage of cybersecurity professionals and qualified software engineers. Moreover, universities are largely moving away from memory-unsafe languages like C in their undergraduate courses. 
+
+All of this demands a shift in mindset: from reactive patching to proactive prevention.
+
+Considering security earlier in the development lifecycle is not a new concept, but certainly no longer optional. This has been recognized in Europe by the passing of the Cyber Resilience Act (CRA) and its security-by-design requirement. 
+
+As the standards for the CRA are currently being developed, it’s important to consider memory safety early in their design, and include strategies to shift towards memory-safe tech where feasible, and include resources for remediation where not.
+
+# Europe Has the Groundwork, but needs to act decisively
+
+For decades, memory unsafe languages like C and C++ were the only viable option for performance-critical systems such as operating systems, browsers, and networking stacks. This is no longer the case.
+
+With the rise of modern, performant memory safe languages—most notably Rust—it is now feasible to build even low-level systems securely. Furthermore, we are seeing encouraging advances towards memory-safe C++. 
+
+That said, we are not advocating a wholesale rewrite of legacy systems. That would not be feasible. But we can and should:
```

Comment:
Cost estimates of this effort can be found in submissions to the U.S. RFP on Open Source Security, if needed.
See #12